src/main/java/org/mule/extension/aggregator/internal/privileged/GroupBasedAggregatorOperationsExecutor.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.aggregator.internal.privileged;
+
+import static org.mule.runtime.module.extension.api.runtime.privileged.ExecutionContextProperties.COMPLETION_CALLBACK_CONTEXT_PARAM;
+import org.mule.extension.aggregator.api.GroupBasedAggregatorParameterGroup;
+import org.mule.extension.aggregator.internal.operations.GroupBasedAggregatorOperations;
+import org.mule.extension.aggregator.internal.routes.AggregationCompleteRoute;
+import org.mule.extension.aggregator.internal.routes.IncrementalAggregationRoute;
+import org.mule.runtime.api.meta.model.operation.OperationModel;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.extension.api.runtime.operation.ComponentExecutor;
+import org.mule.runtime.extension.api.runtime.operation.ExecutionContext;
+import org.mule.runtime.module.extension.api.runtime.privileged.ExecutionContextAdapter;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.reactivestreams.Publisher;
+
+public class GroupBasedAggregatorOperationsExecutor extends GroupBasedAggregatorOperations
```

Comment:
jdoc

Reply:
I see no reason why to extend the GroupBasedAggregatorOperations, those classes should be only placeholders for the model